pkg/maps/lbmap/lbmap.go

```diff
@@ -610,10 +610,15 @@ func (svcs svcMap) addFEnBE(fe *loadbalancer.L3n4AddrID, be *loadbalancer.Backen
 	return &lbsvc
 }
 
-// InitMapInfo updates the map info defaults for sock rev nat {4,6} and lb maps.
+// InitMapInfo updates the map info defaults for sock rev nat {4,6} and lb maps
```

Comment:
I'd probably rename this function to `InitMaps`, and then move all initialization of LB-related maps to this function. Then we would know that at least the maps will be initialized at a single place.

The crux is to ensure that `cilium bpf lb *` will always access a non-nil map.

Reply:
Good suggestion, will do. I'm not sure there's any other approach to simply remembering to initialize the maps. The issue with the CLI code is that it is a separate binary, so it doesn't have access to `option.Config` anyway if a user changes the BPF max size.

Reply:
> Would this actually cause an issue, like the user not seeing all the entries in their BPF maps if they were to cilium bpf lb list, and the daemon was configured with a larger BPF map size?

No, it should not (only the Maglev map, but for it we have a special logic to detect the map size). We just need to make sure that we don't call `bpf.OpenOrCreate()` in the CLI to open maps, as it will recreate the map if it detects a size mismatch.